src/components/common/CategoriesCards/CategoryCard.jsx

```diff
@@ -28,14 +33,16 @@ const CategoryCard = ({
       aria-label={ariaLabel}
       aria-pressed={withAriaPressed ? isActive : undefined}
       style={{
-        backgroundImage: `url(${backgroundImage})`,
+        backgroundImage: `url('${backgroundImage || Stub}')`,
         backgroundSize: 'cover',
         backgroundPosition: 'center',
       }}
     >
       <div className="flex h-16 w-full items-center justify-between self-end rounded-b-lg bg-light-color/80 px-2.5">
         <div className="mt-2 text-start font-fourth text-sm">{title}</div>
-        <div className="fill-current">{icon}</div>
+        <div className="stroke-current">
+          <ErrorBoundary FallbackComponent={FallbackIcon}>{icon}</ErrorBoundary>
```

Comment:
You can optimize usages of ErrorBoundary by moving it to the `Icon` component. In that case you will be sure that you don't miss it.

Here are 2 ways of doing it ether wrap `Icon` with `withErrorBoundary` or create an extra component and use `<ErrorBoundary><Icon .../></ErrorBoundary>`

Reply:
https://github.com/bvaughn/react-error-boundary?tab=readme-ov-file#witherrorboundary-hoc

Reply:
Are you giving me information on how it can be done or telling me how it should be done?

Reply:
The lib provides several options on how you can wrap your component to catch errors. It's more about preference and depends on situation wrap it with either `withErrorBoundary` or `ErrorBoundary`.

Reply:
What you've done with `DynamicIcon` is correct